### PR TITLE
Ccp 125

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCp-125] Fixed review comments for test cases

### CODE REVIEW

File: Conceptpower+Spring/src/test/java/edu/asu/conceptpower/validation/ConceptListAddValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+
+import edu.asu.conceptpower.core.IConceptListManager;
+import edu.asu.conceptpower.web.ConceptListAddForm;
+import edu.asu.conceptpower.wordnet.Constants;
+import junit.framework.Assert;
+
+public class ConceptListAddValidatorTest {
```
Comment (jdamerow) [inline Conceptpower+Spring/src/test/java/edu/asu/conceptpower/validation/ConceptListAddValidatorTest.java]: what about the case that name and description are ok?

**Comment by mkarthik90** [comment]
@jdamerow  Added the success scenario for description and name

**Comment by jdamerow** [comment]
ok to test

File: Conceptpower+Spring/src/test/java/edu/asu/conceptpower/validation/ConceptSearchValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+
+import edu.asu.conceptpower.web.ConceptSearchBean;
+import edu.mit.jwi.item.POS;
```
Comment (jdamerow) [inline Conceptpower+Spring/src/test/java/edu/asu/conceptpower/validation/ConceptSearchValidatorTest.java]: Wrong POS.

**Pull request closed without merging**